src/main/java/com/google/devtools/build/lib/worker/WorkerMultiplexer.java

```diff
@@ -150,7 +150,6 @@ public WorkerProcessStatus getStatus() {
   public synchronized void createSandboxedProcess(
       Path workDir, Set<PathFragment> workerFiles, SandboxInputs inputFiles)
       throws IOException, InterruptedException {
-    // TODO: Make blaze clean remove the workdir.
```

Comment:
Why have you removed this line? Is it already cleaning?

Reply:
I took this from @larsrc-google's CL: src/main/java/com/google/devtools/build/lib/worker/WorkerMultiplexer.java

I guess that with multiplex sandboxing Bazel is responsible for cleaning up the work dirs? Not sure though, I can just leave in the comment if we are uncertain about it.

Reply:
I added it back to be on the safe side.

Reply:
Could you please add this comment back?

Reply:
Done!